src/app/bookshelf/stats/BookshelfStats.tsx

```diff
@@ -0,0 +1,194 @@
+"use client";
+import React, { useEffect, useState } from "react";
+import { GoogleBooksVolumeInfo } from "@/interfaces/googleBooks/GoogleBooks";
+import { useGetBookshelfQuery } from "@/api/bookshelfApi";
+import { Pacifico } from "next/font/google";
+import { BookStatus } from "@/interfaces/BookshelfBook";
+import DotGraph from "@/app/statsGraphs/DotGraph";
+import HorizontalBarGraph from "@/app/statsGraphs/HorizontalBarGraph";
+import PieChartGraph from "@/app/statsGraphs/PieChartGraph";
+import styles from "./BookshelfStats.module.css";
+import Header from "@/components/headers/Header";
+
+const pacifico = Pacifico({
```

Comment:
It's good to have a repository for all the fonts in the project, if you use a few, e.g. having `fonts.ts` file somewhere.